### DISCUSSION TITLE
Sendgrid icon is missing from Paste Icons

### DISCUSSION THREAD

**Discussion by akmittal** [twilio-labs/paste#2843] [Request: Icon] [answered]
It looks like the Sendgrid icon is missing from iconset, Is there any plan to add that?

**Comment by TheSisb** [comment, marked as answer]
Hi @akmittal,

Since the SendGrid logo is full color it won’t render well as a Paste icon. Paste icons are monochrome, so you won't be able to achieve an effect like the official logo.  However. the brand team had contributed the `ProductEmailAPI` icon which has been used as a fill-in for the SendGrid icon. 

You may want to consider either using the logo image if you need full color, or the `ProductEmailAPI` icon.

Hope this helps!
